modules/300-prometheus/hooks/detect_vpa_max.go

```diff
@@ -84,13 +84,26 @@ func calculateNodesCapacity(input *go_hook.HookInput) error {
 	totalPodsMemory := totalPodsCapacity * memoryPerPod
 	totalPodsCPU := totalPodsCapacity * milliCPUPerPod
 
-	// calculate prometheus usage
-	maxMem := resource.NewQuantity(totalPodsMemory/1, resource.BinarySI)
-	maxCPU := resource.NewMilliQuantity(totalPodsCPU/1, resource.DecimalSI)
+	// calculate prometheus usage by MustParse method
+	minMem := resource.MustParse("1000Mi")
+	maxMem := resource.MustParse(fmt.Sprintf("%dMi", totalPodsMemory/1))
```

Comment:
Why have you changed it? MustParse is slower and more dangerous. We can left maxMem and maxCPU as they were